api/v1beta1/disruption_types.go

```diff
@@ -68,6 +69,10 @@ type DisruptionSpec struct {
 	GRPC *GRPCDisruptionSpec `json:"grpc,omitempty"`
 }
 
+//go:embed *
+// EmbededChaosAPI includes the library so it can be statically exported to chaosli
+var EmbededChaosAPI embed.FS
```

Comment:
```suggestion
var EmbeddedChaosAPI embed.FS
```

I think there's a typo (that you may fix at some other places as well).

Reply:
Ah, yes
Fixed